spec/components/devise/omniauth_form_component_spec.rb

```diff
@@ -22,35 +22,31 @@
 
       render_inline component
 
-      expect(page).to have_link "Twitter"
-      expect(page).to have_link count: 1
```

Comment:
Instead of removing this line, we could replace it with:

```ruby
      expect(page).to have_button count: 1
```

This way we still test that the rest of the buttons aren't rendered in this case.

Same in the other tests of this file :wink:.

Reply:
Oki, I just added that.